### ISSUE TITLE
Roundabout in Chrome 19 - Not working on OS X 10.7

### ISSUE THREAD

**Issue by thelau** [fredleblanc/roundabout#34]
On my side, it's causing an aw snap from google chrome since it has been updated in v.19 
I am not sure if the issue reproduces on OS X 10.6. This is unique to Chrome, works well in Safari and others.

You can have a look at my issue here http://pascalmorabito.com/jewellery-joaillerie/ (i try deactivating roundabout and it worked well making it guilty as charged but this is not what i want obviously). Chrome 19 is supposed to be shipped with a new JS engine, so not sure how it affects Roundabout.

Do you have any idea what's the issue?

**Comment by thelau** [comment]
Addendum: Works well with Google Chrome Canary. Problem is specific to Chrome 19.

**Comment by thelau** [comment]
Found the cause at least, if commenting the following lines in roundabout.js (line 490), i don't get the aw snap on chrome, any idea why the last 3 css settings raise a problem:

``` javascript
// update item
child
  .css({
    left: ((factors.x * info.midStage.width + info.nudge.width) - factors.width / 2.0).toFixed(0) + "px",
    top: ((factors.y * info.midStage.height + info.nudge.height) - factors.height / 2.0).toFixed(0) + "px",
    width: factors.width + "px",
    height: factors.height + "px",
    //opacity: (info.opacity.min + (info.opacity.diff * factors.scale)).toFixed(2),
    //zIndex: Math.round(info.zValues.min + (info.zValues.diff * factors.z)),
    //fontSize: (factors.adjustedScale * data.startFontSize).toFixed(1) + "px"
  });
```

Thanks.

**Comment by fredleblanc** [comment]
That's mighty bizarre. Will need to look into why this is happening. Maybe font-size with a decimal is throwing it for a loop?

**Comment by thelau** [comment]
Yes very weird case. A thread is opened on Google Chrome feedback forum but they don't give it much interest.

Font sizes are specified in em in the CSS file which seemed necessary on my side, often with decimals (i just tried removing them on both font-size and line-height without success).

Trying to specifying a fixed font size in px in roundabout.js (line 499) doesn't raise any error so you're right and it seems to be a font size issue.

Any way to use anything else but em with roundabout (as i would have font sizes without decimal in this case)?
